tests/configs/configs_test.go

```diff
@@ -85,4 +90,44 @@ var _ = Describe("Optional Configs", func() {
 			Expect(count).To(BeNumerically(">=", 1))
 		})
 	})
+	When("validating with kubeval", func() {
+		It("should pass", func() {
+			for _, v := range getSupportedK8Versions() {
+				By(fmt.Sprintf("checking version '%s'", v))
```

Comment:
maybe "checking with K8s version.." would be more clear?

Reply:
good point